@xen-orchestra/lite/src/components/pool/network/PoolHostInternalNetworkTable.vue

```diff
@@ -0,0 +1,202 @@
+      <UiQuerySearchBar class="table-query" @search="(value: string) => (searchQuery = value)" />
+      <UiTableActions title="Table actions">
+        <UiButton :left-icon="faEdit" variant="tertiary" accent="info" size="medium">{{ $t('edit') }}</UiButton>
+        <UiButton :left-icon="faTrash" variant="tertiary" accent="danger" size="medium">{{ $t('delete') }}</UiButton>
+      </UiTableActions>
+      <UiTopBottomTable
+        :selected-items="selected.length"
+        :total-items="usableRefs.length"
+        @toggle-select-all="toggleSelect"
+      />
+      <div class="table">
+        <VtsTable vertical-border>
+          <thead>
+            <tr>
+              <template v-for="column of visibleColumns" :key="column.id">
+                <th v-if="column.id === 'checkbox'" class="checkbox">
+                  <UiCheckbox :v-model="areAllSelected" accent="info" @update:model-value="toggleSelect" />
+                </th>
+                <th v-else-if="column.id === 'more'" class="more">
+                  <UiButtonIcon size="small" accent="info" :icon="getHeaderIcon(column.id)" />
+                  {{ column.label }}
+                </th>
+                <ColumnTitle v-else id="networks" :icon="getHeaderIcon(column.id)"> {{ column.label }}</ColumnTitle>
+              </template>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="row of rows" :key="row.id">
+              <td v-for="column of row.visibleColumns" :key="column.id" class="typo p2-regular">
+                <UiCheckbox v-if="column.id === 'checkbox'" v-model="selected" accent="info" :value="row.id" />
+                <!--             NEED TO REMOVE `as XenApiNetwork` -->
+                <div
+                  v-if="column.id === 'name_label' && (row.value as XenApiNetwork).name_label"
+                  v-tooltip="{ placement: 'bottom-end' }"
+                  class="text-ellipsis"
+                >
+                  {{ (row.value as XenApiNetwork).name_label }}
+                </div>
+                <div
+                  v-if="column.id === 'name_description'"
+                  v-tooltip="{ placement: 'bottom-end' }"
+                  class="text-ellipsis"
+                >
+                  {{ (row.value as XenApiNetwork).name_description }}
+                </div>
+                <div v-if="column.id === 'MTU'" v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
+                  {{ (row.value as XenApiNetwork).MTU }}
+                </div>
+                <div
+                  v-if="column.id === 'default_locking_mode'"
+                  v-tooltip="{ placement: 'bottom-end' }"
+                  class="text-ellipsis"
+                >
+                  {{ (row.value as XenApiNetwork).default_locking_mode }}
+                </div>
+                <div v-if="column.id === 'more'">
+                  <VtsIcon accent="info" :icon="faEllipsis" />
+                </div>
+              </td>
```

Comment:
Thanks to the mapping of `useTable`, `column` already contains the value of the cell, so you can do something like this:

```suggestion
              <td v-for="column of row.visibleColumns" :key="column.id" class="typo p2-regular">
                <UiCheckbox v-if="column.id === 'checkbox'" v-model="selected" accent="info" :value="row.id" />
                <VtsIcon v-else-if="column.id === 'more'" accent="info" :icon="faEllipsis" />
                <div
                  v-else
                  v-tooltip="{ placement: 'bottom-end' }"
                  class="text-ellipsis"
                >
                  {{ column.value }}
                </div>
              </td>
```

Another possibility is to remove "checkbox" and "more" columns from column definitions and add them manually:

```suggestion
              <td>
                <UiCheckbox v-model="selected" accent="info" :value="row.id" />
              </td>
              <td v-for="column of row.visibleColumns" :key="column.id" class="typo p2-regular">
                <div v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
                  {{ column.value }}
                </div>
              </td>
              <td>
                <VtsIcon accent="info" :icon="faEllipsis" />
              </td>
```

Reply:
Wonderful !